data-prepper-plugins/obfuscate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/obfuscation/ObfuscationProcessor.java

```diff
@@ -42,6 +42,7 @@ public class ObfuscationProcessor extends AbstractProcessor<Record<Event>, Recor
 
     private final String source;
     private final String target;
+    private final boolean SingleWordOnly;
```

Comment:
Minor: Please use the Java convention of starting field names with a lowercase character.

Thus:

```
    private final boolean singleWordOnly;
```

Reply:
Updated the variable name.